inc/hclib-task.h

```diff
@@ -32,32 +32,20 @@ typedef struct _hclib_task_t {
      * and locality flexible asyncAny
      */
     int is_async_any_type;
```

Comment:
"is_async_any_type" was to support asyncAny interface in HCUPC++. But I am pretty sure it won't work now (same reason that hclib codebase has changed a lot). Once resource worker is in master then we may think of a new design to support asyncAny